doc/en/logging.rst

```diff
@@ -3,21 +3,8 @@
 Logging
 -------
 
-.. versionadded 3.3.0
-
-.. note::
-
-   This feature is a drop-in replacement for the `pytest-catchlog
-   <https://pypi.org/project/pytest-catchlog/>`_ plugin and they will conflict
-   with each other. The backward compatibility API with ``pytest-capturelog``
-   has been dropped when this feature was introduced, so if for that reason you
-   still need ``pytest-catchlog`` you can disable the internal feature by
-   adding to your ``pytest.ini``:
-
-   .. code-block:: ini
-
-       [pytest]
-           addopts=-p no:logging
+.. versionadded:: 3.3
+.. versionchanged:: 3.4
 
 Log messages are captured by default and for each failed test will be shown in
```

Comment:
Note to self: should update this phrasing to mention that pytest will capture WARNING and above only by default.